avs_rvtools_analyzer/templates/analyze.html

```diff
@@ -41,6 +41,69 @@ <h5 class="card-title">{{ risk_name.replace('detect_', '').replace('_', ' ').tit
+</div>
+
+<!-- Azure OpenAI Configuration -->
+<div class="card mb-4" id="azure-openai-config">
+    <div class="card-header">
+        <h4 class="mb-0">
+            <button class="btn btn-link" type="button" data-bs-toggle="collapse" data-bs-target="#collapseAzureOpenAI" aria-expanded="false" aria-controls="collapseAzureOpenAI">
+                <i class="fas fa-robot"></i> AI-Powered Risk Analysis (Optional)
+            </button>
+        </h4>
+    </div>
+    <div id="collapseAzureOpenAI" class="collapse">
+        <div class="card-body">
+            <p class="text-muted">Configure Azure OpenAI to get AI-powered suggestions for migration risks.</p>
+            
+            <!-- Environment Variable Configuration Message -->
+            <div class="alert alert-info env-config-message" role="alert" style="display: none;">
+                <i class="fas fa-cog"></i> <strong>Environment Configuration:</strong> 
+                Azure OpenAI is configured using environment variables. The configuration form below is disabled.
+                <br><small>Deployment: <span id="env-deployment-name"></span></small>
+            </div>
+            
+            <form id="azure-openai-form">
+                <div class="row">
+                    <div class="col-md-6">
+                        <div class="mb-3">
+                            <label for="azureEndpoint" class="form-label">Azure OpenAI Endpoint</label>
+                            <input type="url" class="form-control" id="azureEndpoint" placeholder="https://your-resource.openai.azure.com/">
+                        </div>
+                    </div>
+                    <div class="col-md-4">
+                        <div class="mb-3">
+                            <label for="apiKey" class="form-label">API Key</label>
+                            <input type="password" class="form-control" id="apiKey" placeholder="Your OpenAI API key">
+                        </div>
+                    </div>
+                    <div class="col-md-2">
+                        <div class="mb-3">
+                            <label for="deploymentName" class="form-label">Deployment</label>
+                            <input type="text" class="form-control" id="deploymentName" value="gpt-4" placeholder="gpt-4">
+                        </div>
+                    </div>
+                </div>
+                <div class="d-flex gap-2">
+                    <button type="button" class="btn btn-outline-primary" onclick="testAzureOpenAIConnection()">
+                        <i class="fas fa-plug"></i> Test Connection
+                    </button>
+                    <button type="button" class="btn btn-success" onclick="configureAzureOpenAI()">
+                        <i class="fas fa-check"></i> Configure
+                    </button>
+                </div>
+                <div id="azure-openai-status" class="mt-3"></div>
+            </form>
+        </div>
+    </div>
+</div>
```

Comment:
Nice form but I want to simplify the configuration and keep the AI integration on server side with Env variables. The UI form needs to be removed.